Fix: maj wording mineure

guilletmarion opened the pull request:
Reprise du titre du module didask : https://dinum.didask.com/courses/demonstration/60abc18c075edf000065c987

alanlr reviewed (changes requested):
y a des soucis. Quand tu postules et que tu fermes le formulaire de candidature le bouton est bien remplacé par le texte "vous avez postulé le ..." . Par contre en passant à la fiche suivante, c'est le même message qui est affiché alors qu'aucune candidature n'a été fait pour cette société.

Si on repasse par la liste le problème n'a pas lieu.

guilletmarion committed:
Fix: maj wording mineure

Reprise du titre du module didask : https://dinum.didask.com/courses/demonstration/60abc18c075edf000065c987

kevbarns commented:
:tada: This PR is included in version 1.49.1 :tada:

The release is available on:
- [GitHub release](https://github.com/mission-apprentissage/labonnealternance/releases/tag/v1.49.1)
- `v1.49.1`

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
